src/network/networkd-network-gperf.gperf

```diff
@@ -119,12 +119,13 @@ DHCPServer.EmitTimezone,                config_parse_bool,
 DHCPServer.Timezone,                    config_parse_timezone,                          0,                             offsetof(Network, dhcp_server_timezone)
 DHCPServer.PoolOffset,                  config_parse_uint32,                            0,                             offsetof(Network, dhcp_server_pool_offset)
 DHCPServer.PoolSize,                    config_parse_uint32,                            0,                             offsetof(Network, dhcp_server_pool_size)
-Bridge.Cost,                            config_parse_unsigned,                          0,                             offsetof(Network, cost)
+Bridge.Cost,                            config_parse_uint32,                            0,                             offsetof(Network, cost)
```

Comment:
But I have changed offsetof(Network, cost) in `src/network/networkd-network.h` whilst doing so. No further actions needed here, then?

```
-        unsigned cost;
+        uint32_t cost;
+        uint16_t priority;
```